pkg/regressionallowances/regressions_4.17.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+      "JiraComponent": "kube-apiserver",
+      "TestID": "openshift-tests:d6b41cee7afca1c2a0b52f9e6975425f",
+      "TestName": "[bz-kube-apiserver][invariant] alert/KubeAPIErrorBudgetBurn should not be at or above info",
+      "JiraBug": "https://issues.redhat.com/browse/OCPBUGS-42083",
+      "ReasonToAllowInsteadOfFix": "Degraded near the end of stabilization.  Investigation indicated not a release blocker based on the fact that in our obseved error case this measurement stabilizes under the failure threshold after cluster installation completes.",
```

Comment:
Do we know what happens during installation to produce this problem?  For instance, we know that it isn't only caused by etcd leader loss since on some occasions some requests are still functioning.

Reply:
We do not know that for certain, no.